cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -91,6 +91,39 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
           });
         });
 
+        if (service === 'news') {
```

Comment:
Because we currently do not have AV extended to services such as serbian and ukchina amongst others and also the speed of the tests would be massively impacted

Reply:
> Because we currently do not have AV extended to services such as serbian and ukchina amongst others

In existing tests, we use the check for data to ensure the test runs on services that it should. If additional checks need to be made, then a helper similar to [this](https://github.com/bbc/simorgh/blob/e0d596edc0cf9d0e80f7f4c255a23572d660e24d/cypress/integration/pages/articles/tests.js#L8-L9) is often used.

> and also the speed of the tests would be massively impacted

This test does not make any network requests and checks for the existence of an element within the DOM, which is fast in comparison with some of our other others. 😄 My worry is that excluding tests in this way without good reason leaves a large proportion of our sites untested.

Cc @jamesdonoh

Reply:
When I spoke with @jamesdonoh we originally discussed just applying video tests to `news` to see the impact of time when it came to running each video test against every single service. This is why we were just running it against `news`. I think that it'd be best to push this as is and discuss if we want to approach this differently.

Reply:
From my recollection that discussion was around the tests that the video is actually loading and playing which need to interact with SMP directly and are therefore likely to be the flakiest ones. The guidance being tested is part of our placeholder and should be reliable as any of our usual front-end component tests, so I do not think it needs to be restricted to news, unlike tests that interact with SMP.

Reply:
Had a sync on this and resolved this with having a conditional to check if Media(video) exist in page before checking for the guidance message